MusiciansAPP/MusiciansAPP.BL/Extensions/AlbumsExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MusiciansAPP.Domain;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MusiciansAPP.BL.Extensions;
+
+public static class AlbumsExtensions
+{
+    public static bool IsFullData(
+        this IEnumerable<Album> albums, int pageSize)
+    {
+        return albums.Count() == pageSize
```

Comment:
Let's move similar logic to album, track and artist classes. And then reuse this logic to create one extensions instead of 3 similar ones